docs/_source/docs/resolvers.rst

```diff
@@ -181,6 +248,32 @@ Example:
    parameters:
      VpcIdParameter: !stack_output_external prj-network-vpc::VpcIdOutput prod
 
+
+substitute
```

Comment:
I don't have super-strong feelings about this, but why "substitute" and not "sub" as in CF? I might argue in favor of "sub" for consistency with CF.

Reply:
I don't know. I guess I just don't like the abbreviation. It doesn't seem as clear as `substitute`. @zaro0508 , thoughts on this?

Reply:
I agree "substitute" has more intrinsic clarity, but since most of our users are going to be pretty familiar with CF and `sub` I think we could avoid the mental dichotomy by sticking to CF's nomenclature. But I would like @zaro0508's thoughts as well.

Reply:
I've changed this to `sub`